### DISCUSSION TITLE
Password reset and block inside jelly?

### DISCUSSION THREAD

**Discussion by PutzJimmy** [hrfee/jfa-go#162] [Q&A]
Hi. Let me start off with great work!
When i test the reset password on a user i get the pin popup from jelly but nothing after that seems to happen? (what is really suppose to happen lol)

I had users create safe passwords with symbols, numbers and so but they can still set it to whatever they want inside jelly, is there any way to block or send them to jfa-go password setup again. Seems kinda useless if i have them set up good passwords for them right after be able to change it to "1" :P

**Comment by hrfee** [comment]
For password resets, if you have:
* Password resets enabled
* A method setup to message the user (email, discord, telegram or matrix)
* An address/username connected to their account (user can do this when creating their account, or you can do it yourself in the Accounts tab)
* Have the "Jellyfin directory" set in Settings > Password resets (this should be set to the directory Jellyfin puts password reset files in)

then the user should be sent the PIN. If not, check the jfa-go logs for any evidence of it detecting the password reset. If there isn't anything there, send it on here.

For you other request, it doesn't look like there's any way to restrict the user from changing their password through Jellyfin yet (at least according to the lack of activity on [this feature request](https://features.jellyfin.org/posts/21/restrict-password-changing-for-specific-users)).

**Reply by PutzJimmy** [reply]
Got that part set up right i think, i'll have to test it again once i figure out how to docker-compose jfa-go on my synology (moved my jellyfin server over there from unraid). Also discord server for support ;) ;) ;)
